src/main/java/org/opensearch/security/support/WildcardMatcher.java

```diff
@@ -252,7 +252,7 @@ public boolean matchAny(Collection<String> candidates) {
         return matchAny(candidates.stream());
     }
 
-    public boolean matchAny(String[] candidates) {
```

Comment:
This is being called in [here](https://github.com/opensearch-project/security/blob/12ae0cba4e5bdbe45e85f35078cc785c99cf9ab6/src/main/java/org/opensearch/security/transport/DefaultInterClusterRequestEvaluator.java#L99).

Reply:
Does it break something?

mostly: 
```
matchAny(String[] candidates)
```
==
```
matchAny(String... candidates)
```